vuepress/docs/next/tutorials/getting-started/openshift-install-by-operator.md

```diff
@@ -0,0 +1,152 @@
+---
+sidebarDepth: 2
+---
+# Installation on Red Hat OpenShift using the Operator
+
+## Overview
+The following tutorial shows how to install an Entando Application using the Entando Operator and covers a few common enterprise configurations. You may also find the alternative [manual instructions](./openshift-install.md) useful.
+
+Scenario 1 is similar to the Entando quickstart style of deployment which can be applied in many environments, including on developer laptops. The subsequent scenarios build on that initial setup but can also be performed independently. Unless otherwise noted, you have the freedom to keep or modify the default options when installing the Entando Operator and other resources in the tutorials. 
```

Comment:
"when installing the Entando Operator and other resources in the tutorials" reads like you're installing stuff into tutorials... found/cited/mentioned in the tutorials or rephrase?